client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -97,37 +114,104 @@ export default {
           this.syncedFrame = this.maxFrame;
           return;
         }
+        if (!this.imgs[this.frame].cached || this.loadingVideo) {
+          this.frame -= 1;
+          this.loadingVideo = true;
+          return;
+        }
```

Comment:
A general description of what this function does would be cool.  I don't understand it, but it seems like you do.

Reply:
Description was added for the function and this section as well.